features-json/stream.json

```diff
@@ -277,8 +277,9 @@
       "9.3":"n",
       "10.0-10.2":"n",
       "10.3":"n",
-      "11.0-11.2":"y",
-      "11.3-11.4":"y",
+      "11.0-11.2":"y #3",
+      "11.3":"y",
+      "11.4":"y #3",
```

Comment:
And change 11.4 to 11.3 -11.4
Per Repository rules, no new versions can be added or be splitted into other versions. Please fix that ASAP. Thank you for the time being!!!!

Reply:
Even if only affects 11.4 and not both 11.3 to 11.4, you mustn't break browser versions.

Reply:
Damn, that can get very confusing, but ok.